presto-main/src/main/java/com/facebook/presto/sql/planner/TranslationMap.java

```diff
@@ -122,19 +123,19 @@ public Expression rewrite(FieldOrExpression fieldOrExpression)
         }
     }
 
-    public void put(Expression expression, Symbol symbol)
+    public void put(Expression expression, Symbol symbol, Collection<Expression> columnReferences)
```

Comment:
It looks like this argument is always passed in as `analysis.getColumnReferences()`. There's already an `analysis` field  in `TranslationMap` though, so can't you just use that?